app/components/FormComponents/SelectWithLabelAndMagicAllOption/index.js

```diff
@@ -0,0 +1,86 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import { inputType, metaType } from '../types'
+
+class SelectWithLabelAndMagicAllOption extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      showAllOption: false,
+    }
+  }
+
+  updateDefaulted() {
+    const { showAllOption } = this.state
+    const {
+      input: { value },
+    } = this.props
+    this.state.showAllOption = showAllOption || (true && value)
```

Comment:
(true && value) would this also work? this.state.showAllOption = showAllOption || value

Reply:
Yes it would.